RCode/NEFI/run_bayes_model.R

```diff
@@ -82,22 +78,23 @@ jags.out <- run.jags(model = model,
                      inits=init,
                      monitor = jags_plug_ins$variable.namesout.model)
 
+#6) Save and Process Output; Manually record DIC for now 
+
 write.jagsfile(jags.out, file=file.path("Results/Jags_Models/",paste(site,paste0(model_name,'.txt'), sep = '_')), 
                remove.tags = TRUE, write.data = TRUE, write.inits = TRUE)
 
+png(file=file.path("Results/Jags_Models/",paste(site,paste0(model_name,'_Convergence.png'), sep = '_')))
 plot(jags.out, vars = jags_plug_ins$variable.names.model) 
+dev.off()
 
 jags.out.mcmc <- as.mcmc.list(jags.out)
-
 out <- as.matrix(jags.out.mcmc)
 
 DIC=dic.samples(j.model, n.iter=5000)
 DIC
```

Comment:
@wbeck1990, the easiest way to save this output would be to save it as an RDS file, as that will retain all the mcarray formatting, rather than trying to convert it to a text file or something else. 
Suggested code: 
```
saveRDS(object = DIC, file = file.path("Results/Jags_Models/", paste(site, paste0(model_name,'_DIC.rds'), sep = '_')))
```
Is that what you were hoping to save?